runtime/common/src/lib.rs

```diff
@@ -184,3 +191,50 @@ impl<AssetRegistry: Inspect<AssetId = CurrencyId,Balance = Balance, CustomMetada
 			.ok_or(location)
 	}
 }
+
+
+pub struct OraclePriceGetter<Runtime>(PhantomData<Runtime>);
+impl < Runtime: treasury_buyout_extension::Config + dia_oracle::Config + orml_asset_registry::Config<AssetId = CurrencyId, CustomMetadata = CustomMetadata> > PriceGetter<CurrencyId> for OraclePriceGetter<Runtime> 
+{
```

Comment:
The formatting here looks a bit off. Does the code compile?

Reply:
Yes, it compiles. I added another commit with formatting 👍🏼